main.go

```diff
@@ -98,6 +99,7 @@ func main() {
 			for {
 				prs, resp, _ := client.Search.Issues(
 					// Search query to find PR's
+					context.Background(),
```

Comment:
@shurcooL Got it! Thanks for the explanation. I've changed it to use gin.Context in case I need to use it in the future.